features/support/hooks.rb

```diff
@@ -1,9 +1,18 @@
 Before('@sudo') do
   fail 'sudo authentication failed' unless system 'sudo -v'
   @aruba_timeout_seconds = 15
+  ENV['TREMA_LOG_DIR'] = '.'
+  ENV['TREMA_PID_DIR'] = '.'
+  ENV['TREMA_SOCKET_DIR'] = '.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.